docs/run.md

```diff
@@ -128,12 +128,23 @@ https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage):
 
 4. Run LISA
 
-    Use above `<subscription id>` and `<private key file>` to replace in below command. It may take several minutes to complete.
+    Use above `<subscription id>` and `<private key file>` to replace
+    in commands below. They may take several minutes to complete,
+    depending on the complexity of the passed runbooks. A Marketplace
+    image *must* be provided (by its URN).
 
     ```bash
-    lisa -r ./microsoft/runbook/azure.yml -v subscription_id:<subscription id> -v "admin_private_key_file:<private key file>"
+    lisa -r ./microsoft/runbook/azure.yml -v subscription_id:<subscription id> -v "admin_private_key_file:<private key file>" -v "marketplace_image:Debian:debian-10:10:latest"
     ```
 
+	on Windows' cmd.exe or
```

Comment:
If the user follows the instruction, he will create an alias. So that, it doesn't need to run with lisa.sh. It doesn't need to increase document due to the minor difference.

Reply:
It's a tiny increase, come on. And having someone adding an alias just for you is very invasive.

Reply:
Can you open an issue to me for this document change? The `run.cmd` is very important to new users. It tries to provide an early success as simple as possible. The longer document spends time of every reader.
The people works on both Windows and Linux may try to compare the content to understand difference. When we modify the command line, we need to update two place.

Reply:
I agree things should be simple, and encapsulating a complex poetry env call into those scripts is good enough, isn't it? It's just a matter of destiny that cmd.exe will lookup $PWD for path lookup (and use extensions to derive meaning), that's not a thing in Linux. Having two driver files is perfectly fine (lots of projects have that), whereas having the user install an alias, *which only makes sense inside a certain dir*, is very orthodox. My 2 cents. I can open an issue, what would the wording be about, specifically?

Reply:
Add an issue #1287 . Can you drop this commit?

Reply:
Alright, I'll revisit that later, no biggie